src/lib/Libnet/net_server.c

```diff
@@ -445,12 +429,10 @@ engage_authentication(conn_t *pconn)
 {
 	int	ret;
 	int	sd;
-	char	ebuf[ PBS_MAXHOSTNAME + 1 ];
+	char ebuf[ PBS_MAXHOSTNAME + 1 ];
 
 	if (pconn == NULL || (sd = pconn->cn_sock) <0) {
-
-		log_err(-1, __func__,
-			"Bad arguments, unable to authenticate.");
+		log_err(-1, __func__, "bad arguments, unable to authenticate");
```

Comment:
Why change the message text?

Reply:
I saw most of our error messages (atleast in net_server.c) did not start with Capitals and did not end with a period. Hence to make it similar. However, if you prefer, I can change the log text's back.

Reply:
If the tests pass, I'm fine with the change. I'm simply concerned about regressions. These are the kind of "common sense" changes that should not require an EDD, IMHO.

Reply:
I agree. I do not think any tests depend on these and are mostly usual error messages, so per our updated guidelines should not need an EDD. I will recheck the tests, thanks!